haiku-apps/pdftranslator/pdftranslator-1.0.4.recipe

```diff
@@ -13,24 +13,24 @@ COPYRIGHT="2012-2022 3dEyes**"
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/threedeyes/PDFTranslator/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="0eeae6fcb60269853aa68d521034a866183a6c5856aab5a5e34eea7de21c434f"
+CHECKSUM_SHA256="a10bd69c624d351d2142b2942fc8d64d1ef8606f255e0e9dd53f991ca9abcd4e"
 SOURCE_DIR="PDFTranslator-$portVersion"
 
-ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	pdftranslator$secondaryArchSuffix = $portVersion
 	addon:PDFTranslator$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libmupdf$secondaryArchSuffix
+	lib:libmupdf$secondaryArchSuffix >= 1.19
```

Comment:
Only needed for the developer package, PS, thanks for upstream patch! :+1: 
```diff
- 	lib:libmupdf$secondaryArchSuffix >= 1.19
+ 	lib:libmupdf$secondaryArchSuffix
```

Reply:
No worries. Still working on DocumentViewer. It builds now but gets stuck waiting for a mutex... will investigate further later on.